### DISCUSSION TITLE
plotIPDF only plots circular plots

### DISCUSSION THREAD

**Discussion by Kingtellar** [mtex-toolbox/mtex#1976] [Ask Anything]
Hi, I have the Euler angles for grains (attached) reconstructed from the x-ray diffraction of a FCC material. When I'm trying to plot them in IPF with plotIPDF(ori,yvector,'MarkerSize',5) command, it can only plot in a circular plot like in the attached figure1. What is the issue here and how can I solve this issue and make the plot look right, like in the figure 2? 
![image](https://github.com/mtex-toolbox/mtex/assets/96156375/9f380fc3-c977-4faf-93ac-15038e4cb1ca)
![image](https://github.com/mtex-toolbox/mtex/assets/96156375/75bf2aab-fe70-41e6-967b-ef436346f734)
[EulerAngles_sam3_20.txt](https://github.com/mtex-toolbox/mtex/files/12835660/EulerAngles_sam3_20.txt)

**Comment by ralfHielscher** [comment]
The reason is that you have imported the orientions without the setting crystal symmetry. Have a look here https://mtex-toolbox.github.io/OrientationImport.html how to do it correctly.

Ralf.
